_sass/everything.scss

```diff
@@ -276,17 +394,34 @@ footer {
     }
 
    .headline {
+     margin-bottom: 0;
      .pure-button {
        margin-top: 0;
+       margin-bottom: 50px;
      }
    }
+
+   .nav-menu {
+     padding: 0;
+   }
+
+   .logo {
+     padding: 10px 10px;
+   }
 }
 
 /*
  * -- TABLET (AND UP) MEDIA QUERIES --
  * -- Purecss md (768px) --
  */
-@media screen and (max-width: 48em) {
+@media screen and (min-width: 48em) {
+
+  .pure-u-xl-1-2:first-child > .l-box {
+    padding-left: 0;
```

Comment:
Probably want to add  `padding-left: 0;` to fix this problem:

![example](https://cloud.githubusercontent.com/assets/1840562/23542329/59f86d16-ffa1-11e6-96d3-1ff3d7f1a1c9.png)